docs/developers/dashboard.md

```diff
@@ -2,11 +2,11 @@
 
 The dashboard is written with vanilla JS, CSS and HTML (and love). We made the choice to not used any frameworks like Angular or Vue.js... because dashboard is a very small web site.
 
-This dashboard uses only two externals frameworks, which are Bootstrap and Socket.io.
+This dashboard uses only two externals frameworks, which are TailwindCSS and Socket.io.
 
 Socket.io is useful to dynamically receive events from Copilot and display them.
 
-Bootstrap is a framework very powerful which comes with layout conception (grid to help us with responsively) and components (like modal that we use). But Bootstrap is a very large and heavy framework and we only use a percentage of what it offers. So we made the choice to purge it to reduce loading time on dashboard.
+TailwindCSS is a utility-first CSS framework packed with classes like flex, pt-4, text-center and rotate-90 that can be composed to build any design.
```

Comment:
@Starbeuck I love you update documentation ! Thank you for that.

However it is not obvious why you did this commit that seems good to me.

Can you explain **in your commit message** why you decided to move from Bootstrap to TailwindCSS please ?